### PR TITLE
feat(notifications): Create Transmission DTO and Model

### PR DESCRIPTION
**Pull request by FelixTing**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/.github/Contributing.md.

## What is the new behavior?
Create Transmission DTO and Model for support-notifications V2 API

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]
Replace `TRXESCALATED` with just `ESCALATED`

**Comment by FelixTing** [comment]
Replaced `TRXESCALATED` with `ESCALATED`

File: v2/models/transmissionstatus.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+package models
+
+type TransmissionStatus string
+
+const (
+	Failed       = "FAILED"
+	Sent         = "SENT"
+	Acknowledged = "ACKNOWLEDGED"
+	Trxescalated = "ESCALATED"
```
Comment (cloudxxx8) [inline v2/models/transmissionstatus.go:14]: modify the const name

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**